previews/index.md

```diff
@@ -10,3 +10,9 @@ Before deciding to launch a Preview, a research process is undertaken to determi
 Previews typically run for some period of time and their success is determined by customer adoption. If successful, it will translate into the adoption into the Platform. Previews that are insufficiently used in production will be converted into a community project.
 
 During a Preview, user feedback is central, so the more feedback is received, the better the end result. 
+
+## Previews
```

Comment:
```suggestion
## Current Previews
```

Reply:
The table of previews already has status and the title will repeat the status.
We could change the title to a 'List of previews' if that helps.

Reply:
I'm going to resolve this comment. If this is something that is still a problem, let's review it as a squad (via inbox).